app/views/layouts/_breadcrumbs.erb

```diff
@@ -1,7 +1,9 @@
 <div class="wrap-outer-breadcrumb layout-band">
   <div class="wrap-breadcrumb" role="navigation" aria-label="breadcrumbs">
     <div class="breadcrumbs">
-      <a href="https://libraries.mit.edu" title="MIT Libraries">Home</a>  &raquo;  <a href="/">Thesis Submission</a>
+      <%= link_to('Home', root_path) %>
+      &raquo;
```

Comment:
This breadcrumb file appears to be displaying a navigation path but isn't actually conditional on the user's location. Should it be?

Reply:
Maybe. There are only two legit locations a user can be. The root url or the submission url. I suppose the normal behavior of a breadcrumb is to not display the link from the submission page in this case.

Reply:
will fix

Reply:
Updated to not show the link when on the new thesis page.